stage0_bin/layout.ld

```diff
@@ -86,14 +86,14 @@ SECTIONS {
 
     .bss (NOLOAD) : {
         bss_start = .;
-        *(.bss .bss.*)
+        *(.bss .bss.* .lbss.*)
```

Comment:
Would it be worth adding a comment around here or at the top of the file similar to the content of the commit message in this PR?

Reply:
Dnoe.